packages/accordion/index.with.dependencies.ts

```diff
@@ -0,0 +1,2 @@
+import '../../vendors';
```

Comment:
can we somehow set single entrypoint for multiple scripts? I'm sure that this was passible in webpack 1 see https://webpack.github.io/docs/multiple-entry-points.html